src/main/jbake/content/blog/2017/12/running-agent.adoc

```diff
@@ -0,0 +1,155 @@
+= Running Hawkular Agent
+John Mazzitelli
+2017-12-11
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, agent, prometheus, metrics
+
+Now that Hawkular is moving towards integraton with Prometheus as its metrics collection and storage system, the agent
+has had some changes. This document will discuss those changes and how to run the Hawkular Agent.
+
+First, the Hawkular Agent now only runs as a javaagent (no longer does it run inside WildFly as a subsystem extension).
+So to run the agent, simply attach it as a javaagent to whatever JVM you want to monitor (be it a WildFly or EAP app server or
+any JVM application that exposes metrics via JMX). For example, add this command line option to the command that starts the JVM:
+
+```
+-javaagent:hawkular-javaagent.jar=config=hawkular-javaagent-config.yaml,delay=10
+```
+
+The configuration file contains all the settings necessary to start the agent and connect to the Hawkular Server.
+Once connected, the agent will pull down additional configuration settings from the server.
+
+=== Running in WildFly
+
+If you are attaching the agent to a WildFly server, you must set some additional settings.
```

Comment:
"... you must configure some additional settings."

("set settings" just sounds awkward to me)

Reply:
will change this to 'configure'